### PR TITLE
Update node version to 0.12.9

### PR DESCRIPTION
**Pull request by kgalli**
This change also replaces Henning as a maintainer with Zalando SE.

### CODE REVIEW

**Comment by kgalli** [comment]
@prayerslayer can you have a look at this and merge it. I would really like to update our docker node image asap.

**Comment by prayerslayer** [comment]
Let's see if I get to set up a vagrantbox or something today :+1:

**Comment by kgalli** [comment]
:+1:

**Comment by kgalli** [comment]
@prayerslayer any updates?

**Comment by prayerslayer** [comment]
On it now.

File: Dockerfile [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
-    && curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc" \
-    && gpg --verify SHASUMS256.txt.asc \
-    && grep " node-v$NODE_VERSION-linux-x64.tar.gz\$" SHASUMS256.txt.asc | sha256sum -c - \
-    && tar -xzf "node-v$NODE_VERSION-linux-x64.tar.gz" -C /usr/local --strip-components=1 \
-    && rm "node-v$NODE_VERSION-linux-x64.tar.gz" SHASUMS256.txt.asc \
-    && npm install -g npm@"$NPM_VERSION" \
-    && npm cache clear
+  && curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc" \
+  && gpg --verify SHASUMS256.txt.asc \
+  && grep " node-v$NODE_VERSION-linux-x64.tar.gz\$" SHASUMS256.txt.asc | sha256sum -c - \
```
Comment (prayerslayer) [inline Dockerfile:24]: Can we change `sha256sum -c` to `shasum -a 256 -c`? Do you have `shasum` on your platform?

**Comment by prayerslayer** [comment]
Looks good otherwise. :+1: Unfortunately I can't build this today.

**Comment by kgalli** [comment]
@prayerslayer right now we take the snippet for the RUN command from the official node repository. The `sha256sum` command is available on `debian` and `ubuntu` systems. Our base image is an ubuntu base image. So I would like not to switch to `shasum`. I'm still wondering why you have trouble with this because the command is executed against your base image (which is `ubuntu`).

**Comment by prayerslayer** [comment]
Hm, just tried again and it worked fine, no idea what went wrong last year. I have no objections anymore :)

I will merge your PRs and also update the Ubuntu base image. Images will be available maybe throughout the day, but definitely tomorrow.

Thanks for your patience.

**Comment by kgalli** [comment]
:+1:

**Comment by prayerslayer** [comment]
...as soon as I can actually merge PRs here :grin:

**Comment by kgalli** [comment]
Yes, two maintainers both lacking the right to merge PRs might be an issue ;-) ...

**Comment by prayerslayer** [comment]
Maybe @hjacobs can edit the permissions, as he created the repo?

**Comment by kgalli** [comment]
ping @hjacobs ...

**Pull request merged**